.github/workflows/dashboard_erigon.yml

```diff
@@ -0,0 +1,54 @@
+name: Daily Run Erigon 
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run every day at midnight UTC
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+    types: [ closed ]
```

Comment:
optional: Maybe we should add run on push to master to see all tests passed before the merge?